pkg/securitypolicy/securitypolicy.go

```diff
@@ -14,25 +16,64 @@ const (
 	EnvVarRuleRegex  EnvVarRule = "re2"
 )
 
+// PolicyConfig contains toml or JSON config for security policy.
+type PolicyConfig struct {
+	AllowAll   bool              `json:"allow_all" toml:"allow_all"`
+	Containers []ContainerConfig `json:"containers" toml:"container"`
+}
+
+// AuthConfig contains toml or JSON config for registry authentication.
+type AuthConfig struct {
+	Username string `json:"username" toml:"username"`
+	Password string `json:"password" toml:"password"`
+}
+
+// EnvRuleConfig contains toml or JSON config for environment variable
+// security policy enforcement.
+type EnvRuleConfig struct {
+	Strategy EnvVarRule `json:"strategy" toml:"strategy"`
+	Rule     string     `json:"rule" toml:"rule"`
+}
+
+// ContainerConfig contains toml or JSON config for container described
+// in security policy.
+type ContainerConfig struct {
+	ImageName string     `json:"image_name" toml:"image_name"`
+	Command   []string   `json:"command" toml:"command"`
+	Auth      AuthConfig `json:"auth" toml:"auth"`
+	EnvRules  []EnvRule  `json:"env_rules" toml:"env_rule"`
```

Comment:
Should the toml be env_rules (plural) also?

Reply:
That allows for:

```
[[container.env_rule]]
strategy = "re2"
rule = "PREFIX_.+=.+"
```

otherwise each array entry in the toml would be:

```
[[container.env_rules]]
strategy = "re2"
rule = "PREFIX_.+=.+"
```

which would be somewhat odd and not idiomatic for toml.

Reply:
Oh good point

Reply:
My mind is JSON based

Reply:
to be fair, at some point I had the same question to @SeanTAllen 😄